packages/desktop-client/src/components/accounts/TransactionList.js

```diff
@@ -81,8 +78,7 @@ export default function TransactionList({
   onCloseAddTransaction,
   onCreatePayee
 }) {
-  let dispatch = useDispatch();
-  let table = useRef();
+  // let table = useRef();
```

Comment:
Can this be removed entirely?

Reply:
I'm on the fence about this. It's required if we want to re-enable [these lines](https://github.com/actualbudget/actual/blob/ec1aeef49132ae07833df93a7d396cbbef7db165/packages/desktop-client/src/components/accounts/TransactionList.js#L86-L90). It looks like they offer some sort of scroll-to solution.. But I don't know if it even works.

So perhaps we just remove the already commented out code too? WDYT?

Reply:
Looks like nothing ever sets the `scrollTo` ref, so that code should be ok to remove MO

Reply:
Removed the code :)